doc/api/modules.md

```diff
@@ -6,7 +6,11 @@
 
 <!--name=module-->
 
-In the Node.js module system, each file is treated as a separate module. For
+CommonJS modules are the classic format to package JavaScript code for reuse. If
+you are interested in writing browser-compatible code, check out the
```

Comment:
I think this is disingenuous; ES Modules aren’t browser-compatible unless they omit all non-shimmable core modules, node globals, and also use no bare specifiers (since import maps aren’t a browser standard yet).

Both module systems are almost identically as browser-compatible, with similar caveats.

Reply:
> I think this is disingenuous

I'm guessing you didn't mean it this way, but being "disingenuous" involves intending to mislead.  Let's stick to comments specifically about the text. "I think this is incorrect" communicates the important part without suggesting the author is trying to be deceptive.

Reply:
Oh, my apologies. I meant to convey that it will lead the reader to the wrong conclusions.

Reply:
@ljharb I tried to chose my words carefully, but maybe not enough 😅 what I meant to say if the user is trying to author code that should work on the browser, they should probably be using ESM. Im not sure it’s « wrong », but maybe that’s just an opinion we don’t want to have in our docs. Are you suggesting we leave that part out, or do you have a suggestion to improve the phrasing and remove ambiguity?

Reply:
I think that's an opinion, yes, and one that doesn't actually hold as well as "you should write CJS if you want to target the browser" :-) but we shouldn't need to debate that here.

I'm suggesting that be left out, since it isn't meaningfully better or different regardless of which module format you author, and to be precise, we'd have to list a ton of caveats - for both module systems.

The docs should simply describe the two module systems, and avoid suggesting which one users should use and why.